apps/src/codebridge/hooks/useInitialSources.ts

```diff
@@ -130,12 +135,39 @@ export const useInitialSources = (defaultSources: ProjectSources) => {
       return templateSources || startSources;
     }
     if (isEditingExemplar || isViewingExemplar) {
+      // Show the existing exemplar sources when editing or viewing exemplar sources,
+      // if they exist.
+      if (exemplarSource) {
+        const parsedExemplarSource: ProjectSources = {
+          source: exemplarSource,
+          labConfig: levelStartSources?.labConfig,
+        };
+        if (serializedMaze) {
+          // If there is an existing maze file, we will replace it with the current maze.
```

Comment:
Just confirming my understanding...
If there is an existing maze file [within exemplar sources], we will replace with with the current maze [from level properties].
Is that correct?

Reply:
That's correct! We always want to use the most up to date maze in exemplars.